#
# Temporary hack to provide iconv include directory is well
# (sdl explicitly includes iconv.h)
#
--- sdl-config.in.orig	2012-01-19 06:30:06 UTC
+++ sdl-config.in
@@ -42,7 +42,7 @@ while test $# -gt 0; do
       echo @SDL_VERSION@
       ;;
     --cflags)
-      echo -I@includedir@/SDL @SDL_CFLAGS@
+      echo -I@includedir@/SDL -I@includedir@ @SDL_CFLAGS@
       ;;
 @ENABLE_SHARED_TRUE@    --libs)
 @ENABLE_SHARED_TRUE@      echo -L@libdir@ @SDL_RLD_FLAGS@ @SDL_LIBS@
